localization/pose_twist_fusion_filter/ekf_localizer/package.xml

```diff
@@ -22,6 +22,7 @@
   <test_depend>ament_cmake_gtest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
No need to add `ament_cmake_cppcheck` as it's already part of `ament_lint_common`